src/pages/tasks/TaskAssigneeSelectorModal.js

```diff
@@ -108,6 +108,13 @@ function TaskAssigneeSelectorModal(props) {
         setSearchValue(newSearchTerm);
     };
 
+    const report = useMemo(() => {
+        if (!props.route.params || !props.route.params.reportID) {
+            return null;
+        }
+        return lodashGet(props.reports, `${ONYXKEYS.COLLECTION.REPORT}${props.route.params.reportID}`, undefined);
```

Comment:
I don't think `lodashGet` is necessary here as `props.reports` and `props.route.params` always exist